omod/src/main/webapp/fragments/patientsearch/patientSearchWidget.gsp

```diff
@@ -29,8 +37,11 @@
         // it.birthdateEstimated is of type boolean (doesn't need sanitization)
         birthdateEstimated: ${ it.birthdateEstimated },
         identifier:"${ it.patientIdentifier ? ui.escapeJs(ui.encodeHtmlContent(it.patientIdentifier.identifier)) : '' }"
-    });
-
+    }
+        <% listingAttributeTypeNames.each { attributeName -> %>
+            patientObj["${ attributeName }"] = "${ it.getAttribute(attributeName)?:'' }";
```

Comment:
@dkayiwa I think something like `ui.encodeHtml(String.valueOf(it.getAttribute(attributeName)))` could work, but it's up to @wluyima to make the final call. I have not extensively tested this solution.